appyx-components/backstack/common/src/commonTest/kotlin/android/util/Log.kt

```diff
@@ -0,0 +1,20 @@
+package android.util
+
+import kotlin.jvm.JvmStatic
+
+/**
+ * For unit testing, it is necessary to mock the Android Log class. Attempting
+ * to mock the Logger class, as we have done before, is not feasible due to its
+ * expect tag. This results in compile-time errors as the symbol is duplicated
+ * from the compiler's point of view.
+ */
+class Log {
+
+    companion object {
+        @JvmStatic
+        fun v(tag: String, msg: String): Int = 0
+        @JvmStatic
+        fun d(tag: String, msg: String): Int = 0
+
+    }
+}
```

Comment:
You can enable this setting to have line break in each file

<img width="1235" alt="Screenshot 2023-04-20 at 11 20 05" src="https://user-images.githubusercontent.com/5773436/233339002-2012f79d-79e8-41b8-8cb9-b62e8e32f9d7.png">